quiz/web/vite.config.ts

```diff
@@ -1,11 +1,17 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react-swc";
 import eslint from "vite-eslint-plugin";
+import tsconfigPaths from "vite-tsconfig-paths";
 
 export default defineConfig({
     server: {
         port: parseInt(process.env.VITE_PORT),
         host: "0.0.0.0",
     },
-    plugins: [react(), eslint({ fix: true })],
+    plugins: [
+        react(),
+        tsconfigPaths(),
+        { ...eslint({ failOnError: true, failOnWarning: false }), apply: "build" },
```

Comment:
Updated `eslint` configuration for Vite so it does not break development on error.